src/app/fdctl/config/default.toml

```diff
@@ -280,6 +280,12 @@ dynamic_port_range = "8900-9000"
     # with the `--rpc-pubsub-enable-vote-subscription` argument.
     pubsub_enable_vote_subscription = false
 
+[metrics]
+    # The Solana Labs client supports a SOLANA_METRICS_CONFIG env variable that exports metrics to an InfluxDB instance. 
```

Comment:
Format the comment and wrap like the rest of the file (I think double spaced, 72 chars wide).